README.md

```
@@ -1,36 +1,183 @@
+  - 원하는 날짜에 빠른 약속 추가
+  - 30일 이내의 다가오는 약속 확인
+- 데모 체험 모드
+  - 서비스 사용 전 체험해볼 수 있는 기능
+  - CRUD기능 제한
+
+<br>
+
+## 🔗 협업 프로세스
+
+- ### 기능 단위 작업 관리
+  - 각 기능별 이슈 생성 https://github.com/DnJ0408/saram-byeol/issues
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)